hironx_ros_bridge/rqt_plugin.xml

```diff
@@ -0,0 +1,22 @@
+<library path="src">
+  <class name="HiroNXDashboard" type="hironx_ros_bridge.hironx_dashboard.HiroNXDashboard" base_class_type="rqt_gui_py::Plugin">
+    <description>
+      A Python GUI plugin for displaying a dashboard that displays and interacts with low level hrpsys states.
+    </description>
+    <qtgui>
+      <group>
+        <label>Robot</label>
+        <icon type="theme">folder</icon>
+        <statustip>Plugins related to specific robots.</statustip>
+      </group>
+      <group>
+        <label>Hironx</label>
```

Comment:
oh, so you mean Robot->Hrpsys->Hironx
                                                     ->Nextage
                                               ...


--
◉ Kei Okada


On Wed, Feb 4, 2015 at 2:29 AM, Isaac I.Y. Saito <notifications@github.com>
wrote:

> In hironx_ros_bridge/rqt_plugin.xml
> <https://github.com/start-jsk/rtmros_hironx/pull/317#discussion_r24021758>
> :
>
> > @@ -0,0 +1,22 @@
> > +<library path="src">
> > +  <class name="HiroNXDashboard" type="hironx_ros_bridge.hironx_dashboard.HiroNXDashboard" base_class_type="rqt_gui_py::Plugin">
> > +    <description>
> > +      A Python GUI plugin for displaying a dashboard that displays and interacts with low level hrpsys states.
> > +    </description>
> > +    <qtgui>
> > +      <group>
> > +        <label>Robot</label>
> > +        <icon type="theme">folder</icon>
> > +        <statustip>Plugins related to specific robots.</statustip>
> > +      </group>
> > +      <group>
> > +        <label>Hironx</label>
>
> Do we want to use separate grouping from hrpsys dashboard
> <https://github.com/start-jsk/rtmros_common/blob/c3b86391bac79b04157328f0632c9087e7ba7baf/hrpsys_ros_bridge/rqt_plugin.xml>?
> To me having hrpsys dashboard and this hironx dashboard in the same,
> single group makes more sense, among other plugins for non-hrpsys robots.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/start-jsk/rtmros_hironx/pull/317/files#r24021758>.
>